.pylintrc

```diff
@@ -194,64 +201,33 @@ property-classes=abc.abstractproperty
 # Regular expression matching correct function names
 function-rgx=[a-z_][a-z0-9_]*$
 
-# Naming hint for function names
-function-name-hint=[a-z_][a-z0-9_]*$
```

Comment:
could we add a comment on why all these are being removed (for paper-trail purposes) :)